src/main/java/Duke.java

```diff
@@ -1,10 +1,318 @@
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.lang.reflect.Array;
+import java.security.InvalidParameterException;
+import java.util.MissingFormatArgumentException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+
+
 public class Duke {
-    public static void main(String[] args) {
+
+    private static final String LINE = "___________________________________________________";
+    private static final String BYE_MESSAGE = "Bye. Hope to see you again soon!";
+
+    private static final String COMMAND_BYE = "bye";
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_EVENT = "event";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_MARK = "mark";
+    private static final String COMMAND_UNMARK = "unmark";
+    private static final String COMMAND_DELETE = "delete";
+
+    private static final int REMOVE_FROM = 5;
+    private static final int REMOVE_TO = 3;
+    private static final int REMOVE_BY = 3;
+
+    public static void main(String[] args) throws IOException {
+        File F =  new File("src/main/java/duke.txt");
+        if(!F.exists()){
+            F = new File("duke.txt");
+        }
+        printWelcomeMessage();
+        boolean canPrintFeedback;
+        Scanner in = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<Task>();
+        ArrayList<String> inputCommands = new ArrayList<String>();
+        try {
+            canPrintFeedback = false;
+            loadFileContents(F, tasks,inputCommands,canPrintFeedback);
+        }catch(FileNotFoundException e){
+            System.out.println("File does not exist.");
+        }catch(IOException e){
+            System.out.println("Could not read file.");
+        }
+        while (true) {
+            String userInput = in.nextLine();
+            canPrintFeedback = true;
+            processUserInput(tasks,inputCommands, userInput,F,canPrintFeedback);
+        }
+
+    }
+    private static void writeToFile(String filePath, ArrayList<String>inputCommands) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        for(String s:inputCommands){
+            fw.write(s);
+            fw.write('\n');
+
+        }
+        fw.close();
+    }
+
+    private static void appendToFile(String filePath, String userInput) throws IOException{
+        FileWriter fw = new FileWriter(filePath,true);
+        fw.write(userInput);
+        fw.write('\n');
+        fw.close();
+    }
+
+    private static void loadFileContents(File F, ArrayList<Task>tasks, ArrayList<String>commandInput,
+                                         boolean canPrintFeedback) throws IOException {
```

Comment:
Indentation for wrapped lines should be 8 spaces. See Java coding standards for more info.